core-war/src/main/java/org/silverpeas/web/pdc/control/PdcSearchSessionController.java

```diff
@@ -1094,39 +904,28 @@ private void setExtraInfoToResultsToDisplay(List<GlobalSilverResult> results) {
           titleLink = "javascript:" + markAsReadJS;
         } else if (resultType.equals("Space")) {
           // retour sur l'espace
-          String spaceId = indexEntry.getObjectId();
+          String spaceId = result.getId();
           titleLink = "javascript:" + markAsReadJS + " goToSpace('" + spaceId
               + "');document.location.href='"
               + URLUtil.getSimpleURL(URLUtil.URL_SPACE, spaceId) + "';";
         } else if (resultType.equals("Component")) {
           // retour sur le composant
-          componentId = indexEntry.getObjectId();
-          underLink = URLUtil.getSimpleURL(URLUtil.URL_COMPONENT,
-              componentId);
+          underLink = URLUtil.getSimpleURL(URLUtil.URL_COMPONENT, result.getId());
           titleLink = "javascript:" + markAsReadJS + " jumpToComponent('" + componentId
               + "');document.location.href='" + underLink + "';";
-        } else if (componentId.startsWith("user@")) {
+        } /*else if (componentId.startsWith("user@")) {
           titleLink = URLUtil.getApplicationURL() + URLUtil.getURL(resultType) + indexEntry
               .getPageAndParams();
-        } else if (UserIndexation.OBJECT_TYPE.equals(resultType)) {
-          UserDetail userDetail = getUserDetail(indexEntry.getPK().getObjectId());
-          if (userDetail != null) {
-            result.setThumbnailURL(userDetail.getSmallAvatar());
+        } */else if (UserIndexation.OBJECT_TYPE.equals(resultType)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this "else" on the same line that the previous closing curly brace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)